File: 백준/Bronze/4613. Quicksum/Quicksum.c
#include <stdio.h>

int main() {
    char S[257];

    while (1) {
        fgets(S, sizeof(S), stdin);

        if (S[0] == '#') {
            break;
        }

        int dap = 0;
        for (int i = 0; S[i]; i++) {
            if (S[i] > 64) {
                dap += (S[i] - 64) * (i + 1);           
            }
        }
        printf("%d\n", dap);
    }
    return 0;
}

File: 백준/Bronze/4613. Quicksum/Quicksum.py
import sys
input = lambda: sys.stdin.readline().rstrip()


def main():
    while 1:
        S = input()
        if S == '#':
            break
        dap = 0
        for i in range(len(S)):
            if S[i] == ' ':
                continue
            dap += (ord(S[i])-64)*(i+1)
        print(dap)


if __name__ == '__main__':
    main()